pkg/distributor/ratestore.go

```diff
@@ -0,0 +1,46 @@
+package distributor
```

Comment:
This bit of functionality doesn't need any implementation for the rate store. If we want a no op, it could just return the per-stream-rate-limit. We're only using it when stream sharding is enabled so we should probably just have it return 0